### ISSUE TITLE
(bug) An event description should have the scrollbar visible if the content overflows

### ISSUE THREAD

**Issue by cassidoo** [calcom/cal.com#16260]
### Issue Summary

Right now, if you have an event description that is above ~9 lines of text in the event box, the text overflows. It is a scrollable container, but that is not obvious to the user, and it looks like it's just missing the ending of a sentence. The scrollbar should be visible for scrollable content like this, both for usability and for accessibility.

### Steps to Reproduce

1. (for example) Go to https://cal.com/c/review-my-dev-tool
2. Look at the description of the event on the left
3. Scroll in it

### Expected behavior

The scrollbar should be visible.

### Screenshots

What you see:

<img width="257" alt="image" src="https://github.com/user-attachments/assets/6cc8837d-8753-4040-bb53-d72cb5978415">

What you should see:

<img width="274" alt="image" src="https://github.com/user-attachments/assets/5a8ed5c9-9aa4-4aca-a7da-2fa41ea1e907">

### Environment

- [X] app.cal.com
- [ ] Self-hosted Cal

### Desktop (please complete the following information)

```markdown
- OS: MacOS
- Browser: Edge
- npm: N/A
```

**Comment by arpitmishra61** [comment]
@cassidoo This is the native behavior of browsers. Maybe there is a hack that we can do with CSS.
if you notice, the global scroll bar for the entire page is also not visible on mobile devices only when you scroll it will be visible.
![image](https://github.com/user-attachments/assets/7458e981-0cf3-4dba-8835-b8b8aa3ffe4b)

So we need to make changes specific to the description box so it will not affect at other places

**Comment by cassidoo** [comment]
@arpitmishra61 yeah, for sure. For example, this isn't a problem on Windows. I think it's a bad design decision on Apple's part, but it's worth fixing!

**Comment by shuklamaneesh23** [comment]
Hi @cassidoo,

I'd like to work on this issue. Could you please assign it to me?

Thank you!

**Comment by sameersuman573** [comment]
Hello Mam @cassidoo ,  Mam may I try as This will be my First contribution

**Comment by nta-telmore** [comment]
Hello @cassidoo , this issue reported by me as a Cal user on #16757 but for some reason they closed it as they could not reproduce... I am developer myself and would love to take care of this issue. Will be more than happy to help. 

The solution is to create a nice fade effect at the end of the paragraph with Tailwind classes, so users know that the text continues. Does that make sense?

You can assign this task to me, as I know an immediate solution. Thanks!

**Comment by anikdhabal** [comment]
@nta-telmore We don't assign issues. Feel free to open a PR, as you mentioned that you have an immediate solution. Let's see

**Comment by Devanshusharma2005** [comment]
Closing this issue as its fixed.
